flask_appbuilder/security/manager.py

```diff
@@ -553,6 +590,17 @@ def get_oauth_user_info(self, provider, resp):
                 "id": me["oid"],
                 "username": me["oid"],
             }
+        # for okta
+        elif provider == "okta":
```

Comment:
you should not need to use `elif` here and above, `if` is fine because each if clause always ends in a return statement.

Reply:
This is asking for a refactor also, we could implement a map between the providers and the functions that implement the necessary data mapping. Out of scope for this PR